sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-calculation/e-cert-calculation-process.ts

```diff
@@ -128,12 +139,19 @@ export abstract class ECertCalculationProcess {
               disbursementLog,
             );
             if (!shouldProceed) {
-              // Create notifications to the student and ministry when the disbursement is blocked.
-              await this.createDisbursementBlockedNotifications(
-                eCertDisbursement.studentId,
-                eCertDisbursement.applicationNumber,
-                entityManager,
-              );
+              const shouldCreateNotification =
```

Comment:
When doing the below suggestion, there will just be one method call here to the new service.
https://github.com/bcgov/SIMS/pull/3032/files#r1536263389